xtras/src/send_next.rs

```diff
@@ -0,0 +1,51 @@
+use async_trait::async_trait;
+use tracing::instrument;
+use xtra::message_channel::MessageChannel;
+use xtra::refcount::RefCounter;
+
+/// Arbitrarily hard number allowing flexibility of prioritising messages in actors
+const INTERNAL_MSG_PRIORITY: u32 = 100;
+
+#[async_trait]
+pub trait SendNext<M: Send + 'static, R> {
+    /// Dispatch a prioritised message to an actor without waiting for it to handle it.
+    // If the actor is not connected, it logs a warning.
+    async fn send_next(&self, msg: M);
```

Comment:
I don't really want to participate in a naming discussion but since you asked for it :D 
How about calling it `send_now`?

Reply:
I would probably call it `send_prioritized` - in case we opt for letting the caller decide the priority it could be `send_with_priority`. Don't have a strong opinion about this.

Reply:
None of these names (including the original) tell me that we won't wait for the result. Which I think `send_async_safe` tried to do with the word `async`.

Reply:
based on the feedback above, I'll go with `send_async_next()` (because this encapsulates that we won't wait for result).

Reply:
That sounds like a good name. I was going to write my own comment about this but @luckysori expressed it perfectly